bootstrap/autoload.php

```diff
@@ -0,0 +1,8 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Define some useful constants
+ */
+define('BASE_PATH', dirname(__DIR__));
+define('APP_PATH', BASE_PATH . '/app');
```

Comment:
In general I would like to avoid using these constants at all. Is there any real reason to continue maintain all these constants?

Reply:
Still working on it, thinking how to manage. Needed current approach for test purposes.